assets/schema_phylosearch_input.json

```diff
@@ -0,0 +1,59 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/nf-core/phyloplace/master/assets/schema_input.json",
+    "title": "nf-core/phyloplace pipeline - params.phylosearch_input schema",
+    "description": "Schema for the file provided with params.phylosearch_input",
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "target": {
+                "type": "string",
+                "pattern": "^\\S+$",
+                "errorMessage": "Target name must be provided and cannot contain spaces",
+                "meta": ["id"]
+            },
+            "alignmethod": {
+                "type": "string",
+                "pattern": "^\\S+$",
+                "errorMessage": "Method to use for aligning: 'hmmer' or 'mafft'."
+            },
+            "hmm": {
+                "type": "string",
+                "pattern": "^\\S+.hmm$",
+                "errorMessage": "HMMER HMM file to search sequences with."
+            },
+            "extract_hmm": {
+                "type": "string",
+                "pattern": "^\\S+$",
+                "errorMessage": "Name of HMM file in multi-HMM to extract."
+            },
+            "min_bitscore": {
+                "type": "integer",
+                "errorMessage": "Minimum bitscore for hits to this HMM.",
```

Comment:
is there a number range for this value. e.g. are negative scores allowed? if not, set `minimum` and `maximum` if applicable.